README.md

```diff
@@ -0,0 +1,29 @@
+# Parquet
+This is a package for OpenTAP and features a way to read and write parquet results to the OpenTAP engine.
+
+## Installation
+Write `tap package install Parquet` to install the newest release of the Parquet plugin.
+If you are using OpenTAP in a runner it might already be installed if you are using the latest version.
+
+## Writing results
+Add the result listener called `Parquet` to your test plan.
+These are the options available on the result listener at this moment.
+
+| Name | Description | Default value |
+|-|-|-|
+| File path  | The file path of the parquet file(s). Can use \<ResultType> to have one file per result type. | Results/\<TestPlanName>.\<Date>.parquet |
```

Comment:
Can you list other options or link to the OpenTAP documentation that describes more details about the possible templates?